### PR TITLE
adding pantab and tableauhyperapi recipes

### PR DESCRIPTION
**Pull request by lphexter**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -44,6 +44,7 @@ requirements:
     - pip
```
Comment (synapticarbors) [inline recipes/example/meta.yaml:44]: please do not modify this file. This is the example

File: recipes/pantab/LICENSE.txt [hunk 2]
```diff
@@ -0,0 +1,30 @@
+BSD 3-Clause License
```
Comment (synapticarbors) [inline recipes/pantab/LICENSE.txt:1]: The license file is in the source distribution so you should remove this manually packaged one.

File: recipes/pantab/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - pycparser >=2.20
+    - cffi >=1.14.2
+    - tableauhyperapi
+
+test:
```
Comment (synapticarbors) [inline recipes/pantab/meta.yaml:27]: Please add an import test:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-imports

```yaml
test:
   import:
      - pantab
...
```

File: recipes/tableauhyperapi/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
+{% set name = "tableauhyperapi" %}
+{% set version = "0.0.11556" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-py3-none-manylinux2014_x86_64.whl
```
Comment (synapticarbors) [inline recipes/tableauhyperapi/meta.yaml:9]: conda-forge does not build from `.whl` files. You need a source distribution. See general comment

File: recipes/tableauhyperapi/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
```
Comment (synapticarbors) [inline recipes/tableauhyperapi/meta.yaml:24]: You need an import test as above.

**Review by synapticarbors — CHANGES_REQUESTED** [review]
I am not sure if we can support either of these packages. `pantab` depends on `tablueauhyperapi` and that appears to be a proprietary package with no published source distribution (wheels on pypi only) and a proprietary license, which I'm not sure if conda-forge is willing to accept. 

> 1.1	Grant of License. Subject to all of the terms and conditions of this Agreement, Tableau grants you a non-transferable, non-sublicensable (except as expressly authorized herein), non-exclusive license to (a) use the Software for your own purposes and (b) reproduce and distribute the Software in binary code form only, provided that: (i) you distribute the Software complete and unmodified and only bundled as part of, your Programs (as so bundled, the "Program Package"), (ii) the Programs add significant value to the Software, (iii) you do not distribute additional software intended to replace any component(s) of the Software, (iv) you do not remove or alter any proprietary legends or notices contained in the Software, (v) you do not decompile, disassemble or otherwise attempt to reverse engineer the Software (except to the extent applicable law prohibits such restrictions), (vi) you do not modify or create a derivative work of the Software (except that bundling and distributing as expressly authorized hereunder is permitted), (vii) you do not make any representations, guarantees or warranties of any type with respect to the specifications, features, capabilities or otherwise concerning the Software or on behalf of Tableau, (viii) you only distribute the Software subject to a license agreement that protects Tableau's interests consistent with the terms contained in this Agreement, and (ix) you agree to defend and indemnify Tableau and its licensors from and against any damages, costs, liabilities, settlement amounts and/or expenses (including attorneys' fees) incurred in connection with any claim, lawsuit or action by any third party that arises or results from the use or distribution of any and all Programs and/or Software. "Programs" means one or more software programs which you have independently developed.
> 
> 1.2	Submission to Tableau. Prior to first distribution of the Software to any third party, you shall notify Tableau of the name of your Programs and the applicable Program Package by email to hyperapilicensing@tableau.com. Previously such notifications were sent to licensing@tableausoftware.com. Please note that we will provide limited, and potentially untimely, monitoring for such notifications. You agree that if you elect to distribute the Program Package, Tableau may publicly disclose you as a licensee of the Software and identify your Programs and Program Package. Furthermore, if Tableau reasonably determines that review of your Program Package is necessary for Agreement compliance verification purposes, you will, within ten (10) days of request by Tableau, at no charge, provide Tableau with one (1) fully enabled copy of the Program Package for use by Tableau solely for such purposes.
> 

@conda-forge/staged-recipes -- any thoughts on this?

**Comment by lphexter** [comment]
@synapticarbors yes, I wasn't sure either - if you all have thoughts on whether or not it's possible to add this that'd be great! @conda-forge/staged-recipes

**Comment by xhochy** [comment]
Clause `1.2` sounds like a thing that is hard to achieve, I would say these terms make it unhostable as part of conda-forge.

**Pull request closed without merging**